internal/service/dns_config/model_config_view.go

```diff
@@ -0,0 +1,592 @@
+		MarkdownDescription: `Optional. Specifies a list of client addresses for which AAAA filtering is to be applied.  Defaults to _empty_.`,
+	},
+	"filter_aaaa_on_v4": schema.StringAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             stringdefault.StaticString("no"),
+		MarkdownDescription: `_filter_aaaa_on_v4_ allows named to omit some IPv6 addresses when responding to IPv4 clients.  Allowed values: * _yes_, * _no_, * _break_dnssec_.  Defaults to _no_`,
+	},
+	"forwarders": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: ConfigForwarderResourceSchemaAttributes,
+		},
+		Optional:            true,
+		MarkdownDescription: `Optional. List of forwarders.  Error if empty while _forwarders_only_ or _use_root_forwarders_for_local_resolution_with_b1td_ is _true_. Error if there are items in the list with duplicate addresses.  Defaults to empty.`,
+	},
+	"forwarders_only": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: `Optional. _true_ to only forward.  Defaults to _false_.`,
+	},
+	"gss_tsig_enabled": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: `_gss_tsig_enabled_ enables/disables GSS-TSIG signed dynamic updates.  Defaults to _false_.`,
+	},
+	"id": schema.StringAttribute{
+		Computed:            true,
+		MarkdownDescription: `The resource identifier.`,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.UseStateForUnknown(),
+		},
+	},
+	"inheritance_sources": schema.SingleNestedAttribute{
+		Attributes: ConfigViewInheritanceResourceSchemaAttributes,
+		Optional:   true,
+	},
+	"ip_spaces": schema.ListAttribute{
+		ElementType: types.StringType,
+		Optional:    true,
+		Validators: []validator.List{
+			listvalidator.SizeAtMost(1),
+		},
+		MarkdownDescription: `The resource identifier.`,
+	},
+	"lame_ttl": schema.Int64Attribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             int64default.StaticInt64(600),
+		MarkdownDescription: `Optional. Unused in the current on-prem DNS server implementation.  Unsigned integer, min 0 max 3600 (1h).  Defaults to 600.`,
+	},
+	"match_clients_acl": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: ConfigACLItemResourceSchemaAttributes,
+		},
+		Optional: true,
+		Computed: true,
+		Default: listdefault.StaticValue(types.ListValueMust(types.ObjectType{AttrTypes: ConfigACLItemAttrTypes}, []attr.Value{
+			types.ObjectValueMust(ConfigACLItemAttrTypes, map[string]attr.Value{
+				"access":   types.StringValue("allow"),
+				"acl":      types.StringNull(),
+				"address":  types.StringNull(),
+				"element":  types.StringValue("any"),
+				"tsig_key": types.ObjectNull(ConfigTSIGKeyAttrTypes),
+			}),
```

Comment:
You can use a variable or function to create this, as it is reused by other attributes too

Reply:
Resolved